test/sql/vinyl-opts-cfg.lua

```diff
@@ -5,6 +5,7 @@
 --
 box.cfg {
     vinyl_bloom_fpr = 0.1,
+    vinyl_lookup_cost_coeff = 0.5,
```

Comment:
Why do you set the default value here? I guess it needs to be either dropped or set correspondingly to `vinyl_bloom_fpr` here (e.g., two times greater than the default value).

Reply:
The question is what default value of `lookup_cost_coeff` we should use. I think we should take it from benchmark results: with default value of `lookup_cost_coeff` diff between master version (where `bloom_fpr` always equal 0.05) and changed version RQS must be minimal.